webtool/test_correct.py

```
@@ -829,6 +829,39 @@ def test_ziel_dialekt_auto_nie_dialekt(tmp_path, monkeypatch):
+    projekt.setze_datei("p", "x", sprache="auto")
+
+
+def test_ziel_dialekt_auto_folgt_dem_projekt_standard(tmp_path, monkeypatch):
+    """Spec 10.1: erkanntes `de` + Projekt-Standard `ch` ⇒ `ch`, mit Dialekt-Glaettung.
+
+    Ohne den Vorrang gaebe `von_whisper_code` immer `de` zurueck -- `ch` waere fuer eine
+    `auto`-Datei unerreichbar, obwohl der Nutzer es am Projekt eingestellt hat."""
+    monkeypatch.setenv("TRANSKRIBOR_PROJEKTE", str(tmp_path))
+    _auto_datei(tmp_path, "ch", "de")
+    ziel, dialekt, _ = correct._ziel_dialekt("p", "x")
+    assert "Standarddeutsch" in ziel and dialekt is True
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)